x-pack/plugins/ui_actions_enhanced/public/drilldowns/drilldown_definition.ts

```diff
@@ -99,4 +120,10 @@ export interface DrilldownDefinition<
    * A link where drilldown should navigate on middle click or Ctrl + click.
    */
   getHref?(config: Config, context: ExecutionContext): Promise<string | undefined>;
+
+  /**
+   * List of triggers supported by this drilldown type
+   * This is used in trigger picker when configuring drilldown
+   */
+  supportedTriggers(): SupportedTriggers[];
```

Comment:
If we make this return `string[]`, I guess then we don't need to make all the type changes, e.g. `SupportedTriggers` and `BaseActionFactoryContext`.

Reply:
I guess related to: https://github.com/elastic/kibana/pull/74751#discussion_r469923761